test/e2e/pkg/testnet.go

```diff
@@ -81,8 +81,8 @@ type Testnet struct {
 	Dir  string
 
 	IP               *net.IPNet
-	Validators       map[*Node]int64
-	ValidatorUpdates map[int64]map[*Node]int64
+	Validators       map[string]int64
+	ValidatorUpdates map[int64]map[string]int64
```

Comment:
Same comment as above.

Reply:
`manifest.ValidatorUpdatesMap` is of type `map[string]map[string]int64`, whereas `testnet.ValidatorUpdates` is of type`map[int64]map[string]int64`.
I'm sure there's a way to turn `manifest.ValidatorUpdatesMap` into the same type as `testnet.ValidatorUpdates`, and thus get rid of the latter. But as this is not trivial, I'd rather leave it out of this PR.